KONDUTKODEKSO.md

```diff
@@ -0,0 +1,25 @@
+# Kondutkodekso
```

Comment:
The [article in the Esperanto Wikipedia](https://eo.wikipedia.org/wiki/Kondutkodo) uses "Kondut**kodo**", but "kodekso" is somewhat more precise than just "kodo".